include/fmt/chrono.h

```diff
@@ -1550,11 +1588,11 @@ struct chrono_formatter {
     }
   }
 
-  void write(Rep value, int width) {
+  template <typename RepType> void write(RepType value, int width) {
```

Comment:
Why introduce a template parameter instead of using `Rep`?

Reply:
`Rep` is currently assumed to be an unsigned type. I wanted the proper type to propagate through the code and be deduced in the template. Otherwise there could be an unsigned to signed conversion here which would lead to a -Wconversion error.

Reply:
> `Rep` is currently assumed to be an unsigned type.

I don't think this is correct. `Rep` is the representation type of a duration and it can be signed (in fact it's normally signed).

Reply:
Sorry, lowercase `rep` is used to store the internal duration value, and `rep` is unsigned to avoid overflow. At first I wanted to use the existing logic in the `write()`  function to format a duration value of type lowercase `rep`. This is irrelevant now as I reverted this function to its previous state